packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -180,51 +182,80 @@ export const SubscriptionCreate = ({
             </Localized>
           </div>
 
-          {!hasExistingCard(customer) && paypalScriptLoaded && (
-            <Suspense fallback={<div>Loading...</div>}>
-              <PaypalButton
-                apiClientOverrides={apiClientOverrides}
-                customer={customer}
-                setPaymentError={setPaymentError}
-                idempotencyKey={submitNonce}
-                ButtonBase={paypalButtonBase}
-                currencyCode={selectedPlan.currency}
-              />
-            </Suspense>
-          )}
-
-          <h3 className="billing-title">
-            <Localized id="sub-update-title">
-              <span className="title">Billing Information</span>
-            </Localized>
-          </h3>
+          <div className="subscription-create-pay-with-other">
+            {!hasExistingCard(customer) && paypalScriptLoaded && (
+              <Suspense fallback={<div>Loading...</div>}>
+                <Localized id="pay-with-heading-other">
+                  <p className="pay-with-heading">Select payment option</p>
+                </Localized>
+                <div className="paypal-button">
+                  <PaypalButton
+                    apiClientOverrides={apiClientOverrides}
+                    customer={customer}
+                    setPaymentError={setPaymentError}
+                    idempotencyKey={submitNonce}
+                    ButtonBase={paypalButtonBase}
+                    currencyCode={selectedPlan.currency}
+                  />
+                </div>
+              </Suspense>
+            )}
+          </div>
 
-          {!hasExistingCard(customer) && <AcceptedCards />}
+          <div className="subscription-create-pay-with-card">
+            {!hasExistingCard(customer) && !paypalScriptLoaded && (
```

Comment:
If the customer has an existing subscription paid with PayPal but the script failed to load for whatever reason, they'd get the CC form here I think.

Reply:
No, this block does not control the display of the CC form. That's handled in Payment Form.

I have tried to confirm this by verifying this scenario locally and you're presented with the screen as shown below (with the awkward "last4") as an existing PayPal account consists.

To confirm I tried both disabling paypal between loads and updating the paypalScriptLoaded variable.

Reply:
> No, this block does not control the display of the CC form. ...

Oh right, duh 🤦